### DISCUSSION TITLE
frigate integration not loading on  Home Assistant 2021.12 ?

### DISCUSSION THREAD

**Discussion by peter247** [blakeblackshear/frigate#2442] [General]
just Installed the new version of home assistant and found a notification saying that the frigate integration hadn't loaded.
Which I tried to remove and try to install , but would not delete .
Due to running proxmox with a daily backup I went back to yesterdays copy and made sure it was working before the install .
YES , the frigate integration working before the Home Assistant 2021.12 installation .

**Comment by sinamics** [comment]
https://github.com/blakeblackshear/frigate-hass-integration/releases/tag/v2.2.0

**Reply by peter247** [reply]
Sorry , new to HA , how do I get it to update the integration ?
I've just looked in HACs and there is a new version of frigate card .

i've work it out , but I think I need to remove it first before the upgrade .

**Reply by sunsetitaly** [reply]
Some problem over here, please tell us how to deal with it

**Reply by peter247** [reply]
it was a easy fix for me , I used a backup before the 2021.12 update  
uninstalled the integration.
updated to HA 2012.12.
Went into HACS and updated the frigate integration.
reinstalled the frigate integration.
That was it , but I'm not sure if I had just updated the HACS frigate integration without the above steps if that would have worked also ?.

**Reply by sinamics** [reply]
> Some problem over here, please tell us how to deal with it

I simply upgraded Frigate integration to v2.2 from Hacs menu.
If you dont see the update button, try to reinstall the integration

**Reply by peter247** [reply]
**try to reinstall the integration ?**
You can't remove or reinstall the integration after upgrading to 2012.12 , but you need to be on 2012.12 before you can upgrade the integration.

**Reply by neurocis** [reply]
Is there any upgrade path for this? I do not believe downgrading HA is an option for me.
